waterlog/src/Components/Wastage.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import DailyWastageComponent from './DailyWastage';
+import MonthlyWastageComponent from './MonthlyWastage';
+import SeasonalWastageComponent from './SeasonalWastage';
+
+class Wastage extends Component {
+    constructor(props){
+      super(props);
+
+      this.openGraph = this.openGraph.bind(this);
+      this.state = {
+        display: "daily"
+      }
+    }
+    componentDidMount(){
+      this.openGraph("daily");
+    }
+  
+    openGraph = (cityName) => {
+      var i, tabcontent, tablinks;
+      
+      tabcontent = document.getElementsByClassName("tabcontent");
+      for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+      }
+
+      tablinks = document.getElementsByClassName("tablinks");
+      for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+      }
+    
+      document.getElementById(cityName).style.display = "block";
```

Comment:
Can you also refactor so that you don't need 

`tabcontent = document.getElementsByClassName("tabcontent");`
and
`tablinks = document.getElementsByClassName("tablinks");`
and
`document.getElementById(cityName).style.display = "block";`

Reply:
Refactored